runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/catalog/CatalogHandler.java

```diff
@@ -17,7 +17,7 @@
 
 public interface CatalogHandler
 {
-    int register(String subject, String schema);
+    int register(String subject, String type, String schema);
```

Comment:
Method parameter indentation (all methods).

What are the valid values for `type`?
Is it expected to differ per caller, or at the catalog configuration level?

Reply:
Possible value for type should be `AVRO (default), PROTOBUF, JSON`. But out current implementation only supports `AVRO`